clusterloader2/cmd/clusterloader.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"k8s.io/perf-tests/clusterloader2/pkg/config"
+	"k8s.io/perf-tests/clusterloader2/pkg/framework"
+	"k8s.io/perf-tests/clusterloader2/pkg/test"
+)
+
+var (
+	clientConfigPath = flag.String("clientconfig", "", "path to kuberentes client config")
+	testConfigPath   = flag.String("testconfig", "", "path to test config")
+)
+
+func validateFlags() []error {
+	errList := make([]error, 0)
```

Comment:
Since you seem to already import `k8s.io/apimachinery/pkg/util/errors` pkg, could you please use `errors.Aggregate` in that pkg instead of manually dealing with the list and printing it?

Reply:
Sure. However it looks like this Aggregate is useful only for printing. I don't think it supports append() and len().